doc/languages-frameworks/python.section.md

```diff
@@ -1041,6 +1041,14 @@ buildPythonPackage rec {
     libxslt
   ];
 
+  # tests are meant to be ran "in-place" in the same directory as src
+  doCheck = false;
```

Comment:
Commit message says "enable check phase" :thinking:

Reply:
I guess the test ran in my imaginary machine.🤷